pack/return/rtnotz_encounter.json

```diff
@@ -10,7 +10,7 @@
         "pack_code": "rtnotz",
         "position": 11,
         "quantity": 1,
-        "text": "Perform the setup as indicated in the <i>Night of the Zealot Campaign Guide</i>, with the following exceptions:<br/>- When gathering encounter sets, also gather the new encounter sets for Return to The Gathering shown here.<br/>- Remove Act 1a- \"Trapped\" from the game and replace it with the new Act 1a- \"Mysterious Gateway.\"<br/><b>(Continued on reverse side.)</b>",
+        "text": "Perform the setup as indicated in the <i>Night of the Zealot</i> Campaign Guide, with the following exceptions:<br/>- When gathering encounter sets, also gather the new encounter sets for <i>Return to The Gathering</i> shown here.<br/>- Remove Act 1a—\"Trapped\" from the game and replace it with the new Act 1a— \"Mysterious Gateway.\"<br/><b>(Continued on reverse side.)</b>",
```

Comment:
You can't know this - one of the unwritten style guide rules is to not use `em` dashes in the card text anymore. I think that part of the PR can be reverted.

cc @drawntotheflame we should really document these rules somewhere, maybe even as a linting rule. :p

Reply:
I've amended the commit to not have these changes and noted it for the future. Would definitely appreciate if the unwritten guide got written a little, especially if there's any other style changes like this that I might run afoul! :sweat_smile:

Reply:
I have the following in my notes, also mostly from feedback I received on my PRs. Probably incomplete. We could add this to the README as a start?

- Prefer simple dashes over em dashes.
- Prefer `'` over `’`.
- Prefer [fast] over [free].
- Prefer `<u>` over `<cite>` for marking quotes.
- Always have a space between `[properties]` and surrounding text. (`1 [per_investigator]`, `+1 [willpower]`).

Reply:
I'd have definitely appreciated an even partial style guide over no style guide, so I would be all for it.

Reply:
I'll get to that, thanks for the feedback.